android/src/main/java/com/terveystalo/react_native/matomo_sdk/RNMatomoSdkModule.java

```diff
@@ -24,6 +28,8 @@
 
   private final ReactApplicationContext reactContext;
   private Tracker tracker;
+  private DimensionQueue mOneTimeDimensionQueue;
+  private List<CustomDimension> mCustomDimensions = new ArrayList<>();
```

Comment:
Does the order of the dimensions matter? Would be it be easier to use a map so that additions and removals do not need to iterate over the whole data structure?